myokit/formats/sbml/_importer.py

```diff
@@ -96,53 +102,76 @@ def model(self, path):
             log.warn('Compartments are not supported.')
-        x = dom_child(xmodel, 'listOfSpecies')
+        x = xmodel.find(self.ns['sbml'] + 'listOfSpecies')
         if x:   # pragma: no cover
             log.warn('Species are not supported.')
-        x = dom_child(xmodel, 'listOfConstraints')
+        x = xmodel.find(self.ns['sbml'] + 'listOfConstraints')
         if x:   # pragma: no cover
             log.warn('Constraints are not supported.')
-        x = dom_child(xmodel, 'listOfReactions')
+        x = xmodel.find(self.ns['sbml'] + 'listOfReactions')
         if x:   # pragma: no cover
             log.warn('Reactions are not supported.')
-        x = dom_child(xmodel, 'listOfEvents')
+        x = xmodel.find(self.ns['sbml'] + 'listOfEvents')
         if x:   # pragma: no cover
             log.warn('Events are not supported.')
 
         # Ignore custom functions
-        x = dom_child(xmodel, 'listOfFunctionDefinitions')
+        x = xmodel.find(self.ns['sbml'] + 'listOfFunctionDefinitions')
         if x:   # pragma: no cover
             log.warn('Custom math functions are not (yet) implemented.')
 
         # Parse custom units
-        x = dom_child(xmodel, 'listOfUnitDefinitions')
+        x = xmodel.find(self.ns['sbml'] + 'listOfUnitDefinitions')
         if x:
-            self._parse_units(model, comp, x)
+            self._parse_units(model, comp, x, self.ns['sbml'])
+
+        # add time as independent variable (not explicit in SBML format)
+        if bind_time:
+            # add and bind time variable to component
+            comp.add_variable('time')
+            time = comp.get('time')
+            time.set_binding('time')
+            # set unit and value
+            if 'time' in self.units.keys():
+                unit = self.units['time']
+            else:
+                unit = myokit.units.s
+                log.warn('Unit of time could not be found in file and was by '
+                         + 'default set to seconds.')
+            time.set_unit(unit)
+            time.set_rhs(0.0)
 
         # Parse parameters (constants + parameters)
-        x = dom_child(xmodel, 'listOfParameters')
+        x = xmodel.find(self.ns['sbml'] + 'listOfParameters')
         if x:
-            self._parse_parameters(model, comp, refs, x)
+            self._parse_parameters(model, comp, refs, x, self.ns['sbml'])
 
         # Parse rules (equations)
-        x = dom_child(xmodel, 'listOfRules')
+        x = xmodel.find(self.ns['sbml'] + 'listOfRules')
         if x:
-            self._parse_rules(model, comp, refs, x)
+            self._parse_rules(model, comp, refs, x, self.ns['sbml'])
 
         # Parse extra initial assignments
-        x = dom_child(xmodel, 'listOfInitialAssignments')
+        x = xmodel.find(self.ns['sbml'] + 'listOfInitialAssignments')
         if x:
-            self._parse_initial_assignments(model, comp, refs, x)
+            self._parse_initial_assignments(model,
```

Comment:
You can also just write
```
self._parse_initial_assignments(
    model, comp, refs, x, self.ns['sbml'])
```
I find that's usually more readable :D

Reply:
Ok. Done.